### PR TITLE
refactor: builder projects panel land cards and section

### PR DESCRIPTION
**Pull request by pbosio**

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/MenuSections/SectionLandView.cs [hunk 1]
```diff
@@ -5,14 +5,20 @@
...
 {
+    public event Action OnOpenMarketplaceRequested;
+
     [SerializeField] public LandElementView landElementView;
     [SerializeField] public ScrollRect scrollRect;
     [SerializeField] public GameObject contentContainer;
     [SerializeField] public GameObject emptyContainer;
+    [SerializeField] public GameObject loadingAnimationContainer;
+    [SerializeField] public GameObject noSearchResultContainer;
     [SerializeField] public Button buttonNoLandCTA;
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/MenuSections/SectionLandView.cs:16]: Maybe we should remove `[SerializeField]` or make the fields private. `[SerializeField]` with public members is redundant.

**Review by AjimenezDCL — APPROVED** [review]
Looks good to me!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/LandElementViewShould.cs [hunk 2]
```diff
@@ -25,10 +25,12 @@ public void TriggerCallbacks()
...
 
             void OnCallback(string value) { triggered = true; }
+            void OnCallbackVector2(Vector2Int value) { triggered = true; }
 
-            view.OnEditorPressed += OnCallback;
+            view.OnEditorPressed += OnCallbackVector2;
             view.OnSettingsPressed += OnCallback;
-            view.OnJumpInPressed += OnCallback;
+            view.OnOpenInDappPressed += OnCallback;
+            view.OnJumpInPressed += OnCallbackVector2;
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/LandElementViewShould.cs:33]: Maybe we could use more descriptive names here ( for `OnCallbackVector2` and `OnCallback`)

**Review by sandrade-dcl — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/LandElementViewShould.cs (hunk 2 above)
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/LandElementViewShould.cs:33]: I'm open to suggestions, I can't think in more descriptive names for this callbacks in this test. We are interested in knowing if the callbacks are called in a test called `LandElementViewShould.TriggerCallbacks` 🤷

Comment (sandrade-dcl) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/LandElementViewShould.cs:33]: Ups sorry, I didn't realise it was test code. Forget my message! :)

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/MenuSections/SectionLandView.cs (hunk 1 above)
Comment (pbosio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/MenuSections/SectionLandView.cs]: Made them private and/or internal.
I know that `[SerializeField]` with `public` is redundant, but I like to use `[SerializeField]` like a "tag" telling me "this stuff is set in the gameObject and not by code" 😅
In any case, there were no reason to keep those fields as public. Thanks

**Pull request merged**